### ISSUE TITLE
missing page images

### ISSUE THREAD

**Issue by whanley** [dig-eg-gaz/page-images#2]
@courtneyelon missing all but one page 3; missing 1905-03-27-p1
@caf15b missing 1905-05-10-p8
@Kheyward missing 1905-06-24-p8
@maxconwayfsu missing 1905-06-26-p2

**Comment by whanley** [comment]
Thanks Cassandra.

Still outstanding:
@courtneyelon missing all but one page 3; missing 1905-03-27-p1
@Kheyward missing 1905-06-24-p8
@maxconwayfsu missing 1905-06-26-p2

**Comment by whanley** [comment]
@jeffhayden11 missing 1906-02-05-p5 -- thanks for all the rest!

**Comment by whanley** [comment]
@MarsellaM thanks for all of these--just missing 1906-04-25-p3

**Comment by whanley** [comment]
@madibolton I've arranged your page images, but 1905-03-13-p3 is missing. Was it on the microfilm? As for the supplements, I've simply renumbered them p7 and p8 (even though those page numbers don't appear on the pages themselves). Thanks!

**Comment by madibolton** [comment]
Yes, page 3 was missing from the microfilm for 03/13/1905.
Thank you!

On Apr 18, 2017, at 14:18, Will Hanley <notifications@github.com<mailto:notifications@github.com>> wrote:


@madibolton<https://github.com/madibolton> I've arranged your page images, but 1905-03-13-p3 is missing. Was it on the microfilm? As for the supplements, I've simply renumbered them p7 and p8 (even though those page numbers don't appear on the pages themselves). Thanks!
